includes/rest-api/Controllers/Version2/class-wc-rest-coupons-v2-controller.php

```diff
@@ -511,7 +511,7 @@ public function get_item_schema() {
 							),
 							'value' => array(
 								'description' => __( 'Meta value.', 'woocommerce' ),
-								'type'        => array( 'string', 'null' ),
+								'type'        => 'mixed',
```

Comment:
I wonder if it would be worth adding a comment pointing to `adjust_wp_5_5_datatype_compatibility` or just mention why we're using `mixed` type even though it's not compatible with WP 5.5... but perhaps git blame works as well.

Reply:
Agree, added more explanation in https://github.com/woocommerce/woocommerce/pull/27318/commits/3dd508fd356e44c8799ea7b29227ba04e6df2729.